packages/material-ui/src/Divider/Divider.js

```diff
@@ -99,7 +99,7 @@ Divider.propTypes = {
    */
   component: PropTypes.elementType,
   /**
-   * If `true`, the divider will apply different strategy for height.
+   * If `true`, the divider will apply adapt to a parent flex container.
```

Comment:
```suggestion
If `true`, the divider will adapt to a parent flex container.
```
?

(Fixes the grammar, but it's still not clear what this is actually trying to say.)

Reply:
I'm not sure how we could better explain the role of this prop.

Reply:
We could spell it out 🤷‍♂:
```suggestion
If `true`, a vertical divider will have the correct height when used in flex container. (By default, a vertical divider will have a calculated height of `0px` in Chrome if it is the child of a flex container.)
```